src/components/MonthAndDayDate/MonthAndDayDate.tsx

```diff
@@ -0,0 +1,48 @@
+import "./MonthAndDayDate.css";
+
+const MonthAndDayDate = ({
+  date,
+  size,
+}: {
+  date: Date;
+  size: "small" | "medium" | "large";
+}) => {
+  const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+
+  const formattedTime = `${date.getHours()}:${String(
+    date.getMinutes()
+  ).padStart(2, "0")}`;
+
+  const formattedDate = `${date.getFullYear()}-${String(
+    date.getMonth() + 1
+  ).padStart(2, "0")}-${String(date.getDate()).padStart(2, "0")}`;
+
+  return (
+    <div className={`flex justify-center pt-4 ${size}`}>
+      <p className="mr-1 calendar"> 🗓️ </p>
+      <div className="flex flex-col">
+        <time className="mt-1 date" dateTime={formattedDate}>
+          {date.getDate()}
+        </time>
+        <time dateTime={`${formattedDate}T${formattedTime}`}>
+          {monthNames[date.getMonth()]}
+        </time>
```

Comment:
Now I realized it actually makes more sense to put the `<time>` element instead of this `<div>` and have inside 2 `span`'s. This way, we have just one `<time>` elemenet with the correct `dateTime` of that date.